.ci/install_kata_image.sh

```diff
@@ -13,20 +13,35 @@ set -o errtrace
 cidir=$(dirname "$0")
 rust_agent_repo="github.com/kata-containers/kata-containers"
 arch=$("${cidir}"/kata-arch.sh -d)
+image_path="${image_path:-/usr/share/kata-containers}"
+image_name="${image_name:-kata-containers.img}"
+initrd_name="${initrd_name:-kata-containers-initrd.img}"
+AGENT_INIT="${AGENT_INIT:-no}"
+TEST_INITRD="${TEST_INITRD:-no}"
 
 build_rust_image() {
 	export RUST_AGENT="yes"
 	osbuilder_path="${GOPATH}/src/${rust_agent_repo}/tools/osbuilder"
 	distro="ubuntu"
 
+	sudo mkdir -p "${image_path}"
+
 	pushd "${osbuilder_path}"
-	echo "Building rust image"
-	sudo -E USE_DOCKER=1 DISTRO="${distro}" make -e image
 
-	image_path="/usr/share/kata-containers/"
-	sudo mkdir -p "${image_path}"
-	echo "Install rust image to ${image_path}"
-	sudo install -D "${osbuilder_path}/kata-containers.img" "${image_path}"
+	if [ "${TEST_INITRD}" == "no" ]; then
+		echo "Building image with AGENT_INIT=${AGENT_INIT}"
+		sudo -E USE_DOCKER=1 DISTRO="${distro}" make -e image
+
+		echo "Install image to ${image_path}"
+		sudo install -D "${osbuilder_path}/${image_name}" "${image_path}"
+	else
+		echo "Building initrd with AGENT_INIT=${AGENT_INIT}"
+		sudo -E USE_DOCKER=1 DISTRO="${distro}" make -e initrd
```

Comment:
I though we need `AGENT_INIT=yes` to build initrd. Is that still the case?

Reply:
I don't think systemd support has been dropped, @bergwolf.

Reply:
@fidencio The idea is to add a test to validate running agent as the init process. Right now **ALL** of the CIs are running systemd tests and we have broken the agent-as-init functionality without noticing it.

Reply:
Aha! I see. I didn't understand tis test was supposed to have agent as init. Sorry for the noise.

Reply:
@bergwolf yes we need `AGENT_INIT=yes`. That is set in the jenkins job as an environment variable, which then is taken by this script. 
You can check in the log of the initrd job that it is building with `AGENT_INIT=yes`
```
14:06:29 Install Kata Containers Image
14:06:29 rust image is default for Kata 2.0
14:06:29 /tmp/jenkins/workspace/kata-containers-2.0-tests-ubuntu-initrd-PR/go/src/github.com/kata-containers/kata-containers/tools/osbuilder /tmp/jenkins/workspace/kata-containers-2.0-tests-ubuntu-initrd-PR/go/src/github.com/kata-containers/tests
14:06:29 Building initrd with AGENT_INIT=yes
14:06:29 Creating rootfs for ubuntu
```

http://jenkins.katacontainers.io/job/kata-containers-2.0-tests-ubuntu-initrd-PR/6/consoleFull